src/templates/react/Content/.template.config/template.json

```diff
@@ -0,0 +1,15 @@
+{
+  "author": "Maxime Mangel",
+  "classifications": [ "Fable" ],
+  "name": "Simple elmish application using React as a renderer",
+  "tags": {
+    "language": "F#",
+    "type": "project"
+  },
+  "identity": "Fable.Template.Elmish.React",
+  "groupIdentity": "Fable.Elmish",
+  "shortName": "FableElmishReact",
```

Comment:
Personally I think using `fable-elmish-react` make sense because we will also have `fable-elmish-react-native`.

@et1975 What is your opinion ?